.github/workflows/build.yml

```diff
@@ -20,19 +20,13 @@ jobs:
 
     steps:
       - uses: actions/checkout@v5
-      - name: Set up Python
-        uses: actions/setup-python@v5
-        with:
-          python-version: 3.13
+      - uses: ./.github/actions/setup-python
       - name: Install dependencies
         run: |
           python scripts/os_specific_requirements.py
           python -m pip install --upgrade pip
           pip install -r requirements.txt
         shell: bash
-      - name: Prepare Scripts
-        run: |
-          chmod +x scripts/*
```

Comment:
I think we'd need this if we were calling `./scripts/lint.py` rather than `python scripts/lint.pt`

One less thing to read in this file